test/e2e_tests/specs/CriticalFlow/accountManagement-TC-8639.spec.ts

```diff
@@ -153,7 +154,3 @@ test('Account Management', {tag: ['@TC-8639', '@crit-flow-web']}, async ({pageMa
     //   expect(await components.conversationSidebar().getPersonalUserName()).toContain(member.username);
   });
 });
-
-test.afterAll(async ({api}) => {
```

Comment:
Why we are not cleaning up the teams after the test?

Reply:
The manual cleanup is no longer necessary if the new fixtures like `createTeam`, `createPage` or `createUser` are used, as they will automatically clean up after themselvs after the test finished (including failures etc.).